src/data/themes/index.ts

```diff
@@ -1,22 +1,35 @@
-import type { ImageMetadata } from '@astrojs/image'
+import type { ImageMetadata } from '@astrojs/image/dist/vite-plugin-astro-image.js'
 import data from '../themes.json'
 import type { Link, Theme } from '../../types.js'
 
 interface ThemeData {
     title: string
     description: string
-    image: { src: string; alt: string }
+    fullDescription?: string
+    image: { src: string; alt: string; }
+    images?: { src: string; alt: string; }[]
     repoUrl: Link
     npmUrl?: Link
     demoUrl?: Link
+    links?: Link[]
     categories: string[]
     featured?: number
     slug: string
     stars: number
+    tags?: string[]
```

Comment:
```suggestion
    tags?: string[]
    PREVIEW?: true
```
It might also be a good idea to use a zod schema to parse the theme JSON so it's _definitely_ the shape we expect